api/src/main/java/graphql/nadel/ServiceExecutionHydrationDetails.java

```diff
@@ -0,0 +1,25 @@
+package graphql.nadel;
+
+/**
+ * This contains details about a service hydration call when a {@link ServiceExecution} is invoked.
+ */
+public class ServiceExecutionHydrationDetails {
+    private final int timeout;
+    private final int batchSize;
+
+    public ServiceExecutionHydrationDetails(Integer timeout, Integer batchSize) {
+        this.timeout = timeout == null ? -1 : timeout;
+        this.batchSize = batchSize == null ? 1 : batchSize;
+    }
+
+    /**
+     * @return The timeout in MS to use - a value of -1 indicates no specific timeout was set
+     */
+    public int getTimeout() {
+        return timeout;
+    }
+
+    public int getBatchSize() {
+        return batchSize;
+    }
+}
```

Comment:
A POJO rather than pass in the AST element like we do in other places

Reply:
This will allow us to expand it if we have more details later.  This POJO can grow but the AST representation of the hydration details is an implementation concern.  This "view" is just for invoked services